lib/editing/editor.dart

```diff
@@ -37,6 +38,8 @@ abstract class Editor {
    */
   void execCommand(String name);
 
+  void autoComplete({bool autoInvoked, bool quickFix});
```

Comment:
We should provide default values for these `bool`s; that way we don't have to check `if (autoInvoked != null && autoInvoked)` and similar. And perhaps `autoComplete` ==> `showCompletions`? That way there isn't ambiguity between autoComplete and autoInvoked. Lastly, I think the intent of `quickFix` is something like only show quick fixes? If so, perhaps call it `onlyShowFixes`. This definition may then look something like this:

```
void showCompletions({bool autoInvoked: false, bool onlyShowFixes: false});
```